### PR TITLE
Intel Realsense example

### PR DESCRIPTION
**Pull request by cabelo**
### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [X] I agree to contribute to the project under Apache 2 License.
- [X] To the best of my knowledge, the proposed patch is not based on a code under GPL or other license that is incompatible with OpenCV
- [X] The PR is proposed to proper branch
- [X] There is reference to original bug report and related work
- [X] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [X] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: samples/cpp/videocapture_realsense.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      Mat irImage;
+      Mat adjMap;
+
+      capture.grab();
+      capture.retrieve(depthMap,CAP_INTELPERC_DEPTH_MAP);
+      capture.retrieve(image,CAP_INTELPERC_IMAGE);
+      capture.retrieve(irImage,CAP_INTELPERC_IR_MAP);
+
+      cv::minMaxIdx(depthMap, &min, &max);
+      depthMap.convertTo(adjMap,CV_8UC1, 255 / (max-min), -min);
```
Comment (alalek) [inline samples/cpp/videocapture_realsense.cpp:27]: It is time to learn about `cv::normalize()`.

**Review by alalek — CHANGES_REQUESTED** [review]
There is already `videocapture_intelperc.cpp`.
Why do we need to create one more sample?

File: samples/cpp/videocapture_realsense.cpp (hunk 1 above)
Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp:27]: > 
Hello dear @alalek 

Intel perceptual camera is very incompatible with current realsense. For various reasons and parameters of the videocapture_intelperc example realsense and realsenseid do not work.

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp:27]: > It is time to learn about `cv::normalize()`.

doing...

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: > > It is time to learn about `cv::normalize()`.
> 
> doing...

Thank you @alalek , even if the PR is not approved... I always learn from you!

Hugs.

Comment (alalek) [reply samples/cpp/videocapture_realsense.cpp]: > For various reasons and parameters of the videocapture_intelperc example realsense and realsenseid do not work.

It makes sense to fix the sample and make it compatible with Realsense cameras.

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: done 
opencv

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: Super @alalek  

Not even librealsense is fully compatible with Intel Perceptual (hardware is very obsolete)

I have the Intel family, see below...

realsense

Comment (alalek) [reply samples/cpp/videocapture_realsense.cpp]: If old sample is obsolete, then it makes sense to remove that sample completely and keep the new one.

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: ok

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: done

Comment (alalek) [reply samples/cpp/videocapture_realsense.cpp]: There is reference on removed sample from obsolete tutorial which is patched in previous PR.
So it makes sense to update this too.

Comment (cabelo) [reply samples/cpp/videocapture_realsense.cpp]: Very well remembered. doing...

**Pull request closed without merging**